README.md

```diff
@@ -131,6 +131,18 @@ cd ~/.vim
 git clone https://github.com/vim-ctrlspace/vim-ctrlspace.git .
 ```
 
+#### Neovim
+
+Neovim requires adding the following line to your `.vimrc`:
```

Comment:
This line can be changed to:
>  the following line to your `.vimrc` or `init.vim`:

for those folks that use the XDG_CONFIG method. :)